From: Junchang Wang <junchangwang@gmail.com>
Date: Sun, 30 May 2010 02:26:07 +0000
Subject: [PATCH 13/47] r8169: remove unnecessary cast of readl()'s return
 value

commit 06f555f35fcc917c2b3c32fe151ec370ad47db35 upstream.

readl() returns a 32-bit integer on all platforms.
There is no need to cast its return value.

Signed-off-by: Junchang Wang <junchangwang@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/r8169.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/r8169.c b/drivers/net/r8169.c
index d3de684..31872dd 100644
--- a/drivers/net/r8169.c
+++ b/drivers/net/r8169.c
@@ -89,7 +89,7 @@ static const int multicast_filter_limit = 32;
 #define RTL_W32(reg, val32)	writel ((val32), ioaddr + (reg))
 #define RTL_R8(reg)		readb (ioaddr + (reg))
 #define RTL_R16(reg)		readw (ioaddr + (reg))
-#define RTL_R32(reg)		((unsigned long) readl (ioaddr + (reg)))
+#define RTL_R32(reg)		readl (ioaddr + (reg))
 
 enum mac_version {
 	RTL_GIGA_MAC_NONE   = 0x00,
-- 
1.7.5.4

